app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/template.rb

```diff
@@ -47,7 +51,88 @@ def raw_delete_image
     _log.error("image=[#{name}], error: #{e}")
   end
 
+  def self.raw_import_image(ext_management_system, options = {})
+    session_id = DateTime.now.strftime('%Q')
```

Comment:
Does this session_id have to match some other system?  If you just want a unique id something like `SecureRandom.uuid` would be better

Reply:
done